Code/Mantid/docs/source/interfaces/FilterEventUI.rst

```diff
@@ -0,0 +1,105 @@
+Filter Events Interface
+=======================
+
+.. contents:: Table of Contents
+  :local:
+  
+  
+Description
+-----------
+ 
+The Filter Events interface provides aid to users in order to set up event filter with different `Functionalities <http://docs.mantidproject.org/nightly/algorithms/GenerateEventsFilter-v1.html#functionalities>`_. 
+This interface utilises the algorithm `GenerateEventsFilter <http://docs.mantidproject.org/nightly/algorithms/GenerateEventsFilter-v1.html>`_ 
+which is able to generate event splitters according to user’s requirement for filtering events. The generated time splitters are stored either 
+in a SplittersWorkspace or a MatrixWorkspace. Both of them will be used by algorithm `FilterEvents <http://docs.mantidproject.org/nightly/algorithms/FilterEvents-v1.html#algm-filterevents>`_ 
+which filters the events from a EventWorkspace to one or multiple EventWorkspaces according to an input `SplittersWorkspace <http://www.mantidproject.org/SplittersWorkspace>`_
+containing a series of splitters.
+
+On the bottom section the user has an option of applying `FilterByLogValue <http://docs.mantidproject.org/nightly/algorithms/GenerateEventsFilter-v1.html#generate-event-filters-by-sample-log-value>`_ 
+or `FilterByTime <http://docs.mantidproject.org/nightly/algorithms/GenerateEventsFilter-v1.html#generate-event-filters-by-time>`_ 
+algorithm. User also has an option of filtering events using ``Advanced Setup``.
+
+Layout
+------
+
+.. figure:: ../images/FilterEventsUI.png
+   :align: center
+   :width: 400
+
+
+Data Loading
+------------
+
+For the data loading step, MantidPlot provides an interface in which the user shall select the 
+appropriate file via ``Browse`` button in the ``File/Run`` field. The user must provide a file, which contains valid sample logs 
```

Comment:
I think italics is preferred over monospaced for elements of a UI. (`*italics*`)